packages/nbdime/src/common/mergeview.ts

```diff
@@ -1247,6 +1247,13 @@ export class MergeView extends Panel {
       }
     }
 
+  getMergedValue(): string {
+    if (!this.merge) {
+      throw new Error('No merged value; missing "merged" view');
+    }
+    return this.merge.baseEditorWidget.model.sharedModel.getSource();
```

Comment:
From my test in #12, I think it should be `remoteEditorWidget` otherwise the downloaded merged document is not correct. But if others can checked that point it will be great.

I also used the CM doc content to fit closer the original code and because theoretically we could use a raw editor as the sharedModel is only required due to the usage of juyterlab wrapped code editor.

Reply:
@krassowski Thanks. I will merge this PR.